bumble/profiles/heart_rate_service.py

```diff
@@ -63,7 +63,7 @@ class BodySensorLocation(IntEnum):
     class HeartRateMeasurement:
         def __init__(
             self,
-            heart_rate,
+            heart_rate: int,
```

Comment:
since we're adding types, the other parameters and return value should be typed as well.